components/landingPage.vue

```diff
@@ -0,0 +1,76 @@
+<script>
```

Comment:
Great work! 

Take a look at the [Nuxt Routing](https://v3.nuxtjs.org/getting-started/routing) docs. While we could do it this way, Nuxt comes with routing logic.

I'd suggest creating a `/pages` directory and move this code to `/landing/index.vue`.